joomla/joomla-cms
# [4.1] Workflow Plugin Image Settings

Pull Request for Issue # .

### Summary of Changes
Added new options on the transition settings to set an image of an article (Intro Image + Full Article Image) as required, set a resolution for resizing and set allowed file extensions.

![image](https://user-images.githubusercontent.com/30879672/90650222-e9b32100-e23b-11ea-9b3c-b90b6e25d986.png)



### Testing Instructions
1. Navigate to Content -> Workflow -> Transitions
2. Edit a transition of your choise
3. Go to the tab "Image Settings", choose your settings and save
4. Navigate to Content -> Articles and execute the edited transition
5. Make sure, the transition is not executed, if one of the images, which is set to be required, is missing
6. Make sure, the transition is not executed, if one of the images has a not-supported file extension
7. Make sure, there is a resized image in the image folder with the correct resolution
